internal/util-logging/src/main/scala/sbt/util/LoggerContext.scala

```diff
@@ -181,12 +181,11 @@ object LoggerContext {
         case l    => l.clearAppenders()
       }
     }
-    def close(): Unit = {
-      closed.set(true)
+    def close(): Unit = if (closed.compareAndSet(false, true)) {
```

Comment:
Just a small enhancement to make sure `close()` runs only once, like in line 110 of this file.